### DISCUSSION TITLE
Sh-esp32

### DISCUSSION THREAD

**Discussion by psm12** [hatlabs/discussions#68] [General]
Hi, just starting to use my new sh-esp.

Here are a few comments/suggestions.

1/ the documentation tab doesn't appear on the top level hatlabs.fi website.

2/ the NMEA2000 example/tutorial requires 12v power on the NMEA interface to prevent continuous rebooting (not in the doc but I'm sure I saw it in an earlier discussion)

3/ the Wi-Fi interface doesn't seem to be enabled in the NMEA example code. 

4/ the one-wire example/tutorial requires the password thisisfine to configure the sh on a Wi-Fi network (I dug this out of the SensESP documentation) it would be good to reference this.

I hope the above is helpful.

**Comment by mairas** [comment]
Thank you! Highly appreciated! I'll fix these issues in the near future.
